README.md

```diff
@@ -114,98 +82,124 @@ there:
 
        ./dependencies/down.sh
 
+### Running
+
 1. Start the Lighthouse service specified in the `docker-compose.yml` from the
    root of the repository:
 
         docker-compose up
 
-1. Create a `.env` file which contains the line:
+1. Find the lighthouse container id with:
 
-        LOCALHOST=host.docker.internal
+        docker ps
 
-1. Start the docker container and open a bash session in it with:
+1. Start a bash session in the container with:
 
-        docker run --env-file .env -p 8000:8000 -v $(pwd):/code -it --entrypoint bash lighthouse:develop
+        docker exec -ti <container_id for lighthouse> bash
```

Comment:
You shouldn't need to find out the container ID as Docker Compose names the container as well.  It will be called `lighthouse_lighthouse1` which you should be able to use in place of the container ID for this command.

```suggestion
        docker exec -ti lighthouse_lighthouse1 bash
```

Reply:
Good idea thanks. Mine had an underscore before the 1, hopefully that's consistent.

Reply:
Yes sorry I did mean to include an underscore before the 1.  I'm starting to think the naming is not consistent over time though!  I have the newest version of Docker Desktop installed a couple days ago and now my container has the name `lighthouse-lighthouse-1` with hyphens instead of underscores.

Reply:
Uh oh! I'll add a disclaimer